### PR TITLE
Fix Links

### PR DESCRIPTION
**Pull request by alfakini**

### CODE REVIEW

File: src/__tests__/components/__snapshots__/Navbar.test.tsx.snap [hunk 1]
```diff
@@ -99,44 +93,54 @@ exports[`NavBar renders it with showSocialIcons 1`] = `
...
+          alt="Element Logo"
           src="element-logo.svg"
         />
-      </span>
+      </a>
+       
       <a
-        className="navbar-brand"
+        className="sc-dlnjwi"
         href="https://twitter.com/network"
```
Comment (laurogripa) [inline src/__tests__/components/__snapshots__/Navbar.test.tsx.snap:132]:
```suggestion
        href="https://twitter.com/kusamanetwork"
```

**Review by laurogripa — CHANGES_REQUESTED** [review]
Not sure why, but spacing is looking weird and Connect Wallet button is not working anymore

Previous: https://dreamy-gates-ad44c0.netlify.app/

Now: https://deploy-preview-36--dreamy-gates-ad44c0.netlify.app/

**Comment by murilooon** [comment]
image

Cyborg Guide Button css broked :cry:

**Comment by alfakini** [comment]
[Avoid connecting to wallet when navigating pages](https://trello.com/c/vpUVfYfE/235-avoid-connecting-to-wallet-when-navigating-pages)

**Review by laurogripa — APPROVED** [review]

**Comment by laurogripa** [comment]
> Not sure why, but spacing is looking weird and Connect Wallet button is not working anymore
> 
> Previous: https://dreamy-gates-ad44c0.netlify.app/
> 
> Now: https://deploy-preview-36--dreamy-gates-ad44c0.netlify.app/

Nevermind, the problem was already happening. Opened a card to address this: https://trello.com/c/uQyNPyxG/237-connect-wallet-button-is-irresponsive-if-the-user-closes-the-pop-up

**Pull request merged**